yaml/builders/image-build-push.yaml

```diff
@@ -28,7 +28,7 @@
             set -ex; cd sources;
 
             image_id=$(cat output | grep -- '-> Tagging image' | cut -d' ' -f 4 | sed "s/'//g")
-            image_tags=$(docker image inspect "$image_id" --format '{{range $i, $tag := .RepoTags}}{{$tag}} {{end}}')
+            image_tags=$(docker image inspect $image_id --format '{{range $i, $tag := .RepoTags}}{{$tag}} {{end}}')
```

Comment:
Here should be though only  a single image, no?

Reply:
If not, we should **iterate** over a list of image ids (which would be better called `$imgae_ids`).

Reply:
>Here should be though only a single image, no?

Depends. If we are [tagging multiple versions](https://gist.github.com/rhscl-bot/ecfe6604430d075037a3a0a1ff95955a) of an image it picks up all of them in the grep above.

>If not, we should iterate over a list of image ids

Agreed, will rewrite into a loop.